ortools/linear_solver/xpress_interface.cc

```diff
@@ -382,16 +414,11 @@ std::string XpressInterface::SolverVersion() const {
   int version = 0;
   CHECK_STATUS(XPRSgetintcontrol(mLp, XPRS_VERSION, &version));
 
-  int const major = version / 1000000;
-  version -= major * 1000000;
-  int const release = version / 10000;
-  version -= release * 10000;
-  int const mod = version / 100;
-  version -= mod * 100;
-  int const fix = version;
+  int const major = version / 100;
+  version -= major * 100;
+  int const release = version;
```

Comment:
It seems that [you're right](https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/VERSION.html). Did the previous implementation return something like "XPRESS library version 0.00.00.xx" ?